sdk/nodejs/resource.ts

```diff
@@ -461,6 +466,10 @@ export interface ResourceOptions {
      * An optional existing ID to load, rather than create.
      */
     id?: Input<ID>;
+    /**
+     * An optional existing URN to load, rather than create.
+     */
+    urn?: Input<URN>;
```

Comment:
consider stating that this is incompatible with `id` (and vice versa) and adding checks/asserts.

Reply:
Done.